packages/node/src/events/propose-install-virtual/controller.ts

```diff
@@ -21,7 +21,7 @@ export default async function proposeInstallVirtualEventController(
     requestHandler.store,
     nodeMsg.data.params,
     nodeMsg.data.appInstanceId,
-    nodeMsg.data.initiatingAddress,
+    nodeMsg.data.proposedByAddress,
```

Comment:
Let's stay consistent with the `Identifier` suffix.

Reply:
Oh actually this is a bug. Fixing.